### DISCUSSION TITLE
Follow the file location, when file location changes + add exception measures

### DISCUSSION THREAD

**Discussion by krstp** [zed-industries/zed#45713] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Two things:
- Currently ZED does not allow save to the file that during edit changed its location. Any additional changes to file are saved... not sure where, but definitely not at the old path and definitely not in the new path; any save that happens must go to some kind of temporary location. As currently new input does not seem to be saved anywhere, there should at least some kind of exception prompt that informs the file is not saved or is saved to some temp location.
- Give ZED ability to keep track of the file location, as such, when the file is `moved` or `dragged` to a new location, ZED shall be able to follow the file path or at least save new changes to the "old path", however old path is not optimal.

**Comment by csz5024** [comment]
+1
